### PR TITLE
OSDOCS-5049c: Replacing "whitelist" and "blacklist" references to "allow" and "deny" in route-related docs

### PR DESCRIPTION
**Pull request by DCChadwick**
Version(s): 4.18

Issue: [OSDOCS-5049](https://issues.redhat.com//browse/OSDOCS-5049)

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by DCChadwick** [comment]
@lihongan @rhamini3 - This PR is ready for QE review.

**Review by Miciah — COMMENTED** [review]
One minor suggestion, but
/lgtm

File: modules/nw-route-specific-annotations.adoc [hunk 1]
```diff
@@ -68,7 +68,7 @@ This value is applicable to re-encrypt and edge routes only. For more informatio
 |===
 [.small]
 --
-1. If the number of IP addresses and CIDR ranges in an allowlist exceeds 61, they are written into a separate file that is then referenced from `haproxy.config`. This file is stored in the `var/lib/haproxy/router/whitelists` folder.
+1. If the number of IP addresses and CIDR ranges in an allowlist exceeds 61, they are written into a separate file that is then referenced from `haproxy.config`. This file is stored in the `var/lib/haproxy/router/allowlists` folder.
```
[the remark this answers is not in the document]
Comment (DCChadwick) [reply modules/nw-route-specific-annotations.adoc:71]: Thanks for pointing this out. I'll look into it.

File: modules/nw-route-specific-annotations.adoc [hunk 2]
```diff
@@ -38,7 +38,7 @@ Note: Using this annotation provides basic protection against denial-of-service
...
 |`ingresses.config/cluster ingress.operator.openshift.io/hard-stop-after` | You can set either an IngressController or the ingress config . This annotation redeploys the router and configures the HA proxy to emit the haproxy `hard-stop-after` global option, which defines the maximum time allowed to perform a clean soft-stop. | `ROUTER_HARD_STOP_AFTER`
 |`router.openshift.io/haproxy.health.check.interval`| Sets the interval for the back-end health checks. (TimeUnits) | `ROUTER_BACKEND_CHECK_INTERVAL`
-|`haproxy.router.openshift.io/ip_whitelist`
+|`haproxy.router.openshift.io/ip_allowlist`
 | Sets an allowlist for the route. The allowlist is a space-separated list of IP addresses and CIDR ranges for the approved source addresses. Requests from IP addresses that are not in the allowlist are dropped.
```
Comment (aspauldi) [inline modules/nw-route-specific-annotations.adoc:42]: Is the list named allowlist or allow list? I see it is described as allowlist here but in line 12 it is named allow list.

**Comment by aspauldi** [comment]
Hi Daniel, This looks good! I noticed only a small inconsistency in the spelling of allow list (or allowlist). 

One other nit that I noticed across **Route configuration** as a whole is that Ingress Controller also appears as ingress controller here and there. I know this inconcistency is outside the scope of your changes, but I wanted to mention it anyway.

File: modules/nw-route-specific-annotations.adoc (hunk 2 above)
Comment (DCChadwick) [reply modules/nw-route-specific-annotations.adoc:42]: Thanks! From what I understand, it is supposed to be "allow list" in the running text but the annotation itself is ip_allowlist.

**Comment by DCChadwick** [comment]
@aspauldi Thanks and good catch! I'll update those references to the ingress controller and re-push now.

**Pull request merged**